src/com/connectsdk/service/capability/MediaControl.java

```diff
@@ -37,8 +37,15 @@
     public final static String PlayState_Subscribe = "MediaControl.PlayState.Subscribe";
     public final static String Position = "MediaControl.Position";
 
+    /**
+     * This capability is deprecated. Use PlaylistControl instead.
```

Comment:
`PlaylistControl` is a class name. Shouldn't it be "…Use PlaylistControl.Previous instead."?